internal/uvm/scsi.go

```diff
@@ -195,95 +210,42 @@ func (uvm *UtilityVM) AddSCSI(ctx context.Context, hostPath string, uvmPath stri
 //
 // `readOnly` set to `true` if the physical disk should be attached read only.
 func (uvm *UtilityVM) AddSCSIPhysicalDisk(ctx context.Context, hostPath, uvmPath string, readOnly bool) (*SCSIMount, error) {
-	return uvm.addSCSIActual(ctx, hostPath, uvmPath, "PassThru", false, readOnly)
-}
-
-// AddSCSILayer adds a read-only layer disk to a utility VM at the next
-// available location and returns the path in the UVM where the layer was
-// mounted. This function is used by LCOW as an alternate to PMEM for large
-// layers.
-func (uvm *UtilityVM) AddSCSILayer(ctx context.Context, hostPath string) (*SCSIMount, error) {
-	if uvm.operatingSystem == "windows" {
-		return nil, ErrSCSILayerWCOWUnsupported
-	}
-
-	sm, err := uvm.addSCSIActual(ctx, hostPath, "", "VirtualDisk", true, true)
-	if err != nil {
-		return nil, err
-	}
-
-	if sm.UVMPath != "" {
-		return sm, nil
-	}
-	sm.UVMPath = fmt.Sprintf(lcowSCSILayerFmt, sm.Controller, sm.LUN)
-	return sm, nil
+	return uvm.addSCSIActual(ctx, hostPath, uvmPath, "PassThru", readOnly, VMAccessTypeIndividual)
 }
 
 // addSCSIActual is the implementation behind the external functions AddSCSI and
-// AddSCSILayer.
+// addSCSIActual.
```

Comment:
addSCSIActual is the implementation behind addSCSIActual?

Reply:
LOL. Fixed.